tests/acceptance/features/apiOcm/share.feature

```diff
@@ -611,3 +611,90 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-10222
+  Scenario: user lists permissions of a resource shared to a federated user
+    Given using server "LOCAL"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And user "Brian" has uploaded file with content "ocm test" to "/textfile.txt"
+    And user "Brian" has created folder "folderToShare"
+    And user "Brian" has sent the following resource share invitation to federated user:
+      | resource        | folderToShare                 |
+      | space           | Personal                      |
+      | sharee          | Alice                         |
+      | shareType       | user                          |
+      | permissionsRole | Viewer                        |
+      | federatedServer | @federation-ocis-server:10200 |
```

Comment:
We don't need to specifiy the `federatedServer` in table as it was not used in the requests but used only for logical purpose previously
```suggestion
      | shareType       | user                          |
      | permissionsRole | Viewer                        |
```